src/index.html

```diff
@@ -5,8 +5,344 @@
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+        </div>
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__text">Reviews: 5</p>
+
+        </div>
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button class="button" data-qa="card-hover">
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
+        <div class="card__image-wrapper">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+        </div>
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__text">Reviews: 5</p>
+
+        </div>
+        <div class="card__price">
+
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button class="button" data-qa="hover">
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
+        <div class="card__image-wrapper">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+        </div>
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__text">Reviews: 5</p>
+
+        </div>
+        <div class="card__price">
+
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-value">$2,199</p>
+        </div>
```

Comment:
Fix code style everywhere
```suggestion
        <div class="card__price">
          <p class="card__price-label">Price:</p>
          <p class="card__price-value">$2,199</p>
        </div>
```